src/seedsigner/models/psbt_parser.py

```diff
@@ -79,6 +79,9 @@ def parse(self):
 
         self._set_root()
 
+        # Try to fix zero fingerprints before parsing
```

Comment:
I would add an explanation here describing the issue (just a sentence or two, if possible). Probably also a good idea to point to the github Issue so that the full context and detail can be provided there.